src/ducks/balance/components/BalanceHeader.styl

```diff
@@ -12,3 +12,18 @@
         flex-basis 159px
         padding-top: .5rem
         padding-bottom 0
+
+.SearchIcon
+    position absolute
+    right 2rem
+    color white
+    z-index 1
```

Comment:
Would it make sense to use the z-index scale from cozy-ui here?

Reply:
No, I don't think so. The z-index is very local here, just to say that the search icon is above the rest of the current container.